### PR TITLE
Changed vercel sub-path to no-fs

### PR DESCRIPTION
**Pull request by ryan-blunden**
Changed the exports sub-path from `vercel` to `no-fs` as to be more generic, as well as only allow the sub-path variant to be used in the generated module files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,9 +31,9 @@
       "require": "./src/index.js",
       "import": "./src/index.js"
     },
-    "./vercel": {
-      "require": "./src/vercel.js",
-      "import": "./src/vercel.js"
+    "./no-fs": {
```
Comment (Piccirello) [inline package.json:34]: Are there tests specifically for no-fs? We'll want to ensure it doesn't break since it's exported.

Comment (ryan-blunden) [reply package.json:34]: Yep, there are 3 `secrets.build` related tests that consume the generated JS module.

Comment (Piccirello) [reply package.json:34]: Those appear to use `index.js` rather than `no-fs` though. We'll want to test the no-fs import explicitly.

Comment (ryan-blunden) [reply package.json:34]: See https://github.com/DopplerHQ/gitops-secrets-nodejs/pull/18

**Pull request merged**